cibyl/orchestrator.py

```diff
@@ -68,7 +68,8 @@ def __init__(self, environments: list = None):
         """Orchestrator constructor method"""
         self.parser = Parser()
         self.config = AppConfig()
-        self.publisher = Publisher()
+        # publisher will be initialized when needed
+        self.publisher = None
```

Comment:
You may remove this variable, the publisher is only used in one function.

Reply:
True, I'll remove it.